configs/default/TMTR-TMOTORF7_AIO.config

```diff
@@ -0,0 +1,127 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Aug 28 2019 / 06:11:37 (4f923e482) MSP API: 1.42
+
+#define USE_GYRO_SPI_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define USE_FLASH_M25Q128
+#define USE_BARO_SPI_BMP280
+#define USE_BARO_BMP280
+#define USE_MAX7456E
+
+board_name TMOTORF7_AIO
+manufacturer_id TMTR
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 C06
+resource MOTOR 2 C07
+resource MOTOR 3 B00
+resource MOTOR 4 B01
+resource MOTOR 5 B06
+resource MOTOR 6 B08
+resource MOTOR 7 B07
+resource MOTOR 8 C09
+resource PPM 1 A15
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource I2C_SCL 2 B10
+resource I2C_SDA 2 B11
+resource LED 1 B02
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 B09
+resource ADC_BATT 1 C01
+resource ADC_RSSI 1 C03
+resource ADC_CURR 1 C02
+resource BARO_CS 1 C15
+resource FLASH_CS 1 C08
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource PINIO 1 C14
+
+# timer
+timer C06 AF2
+# pin C06: TIM3 CH1 (AF2)
+timer C07 AF2
+# pin C07: TIM3 CH2 (AF2)
+timer B00 AF3
+# pin B00: TIM8 CH2N (AF3)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+timer B09 AF3
+# pin B09: TIM11 CH1 (AF3)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin C06 0
+# pin C06: DMA1 Stream 4 Channel 5
+dma pin C07 0
+# pin C07: DMA1 Stream 5 Channel 5
+dma pin B00 0
+# pin B00: DMA2 Stream 2 Channel 0
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin A15 0
+# pin A15: DMA1 Stream 5 Channel 3
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+
+# feature
+feature OSD
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set baro_spi_device = 3
+set blackbox_device = SPIFLASH
+set current_meter = ADC
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set osd_gps_sats_show_hdop = ON
+set max7456_spi_bus = 2
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set align_board_yaw = -45
+set pinio_box = 40,255,255,255
+
+
```

Comment:
Please remove one line

Reply:
Missing `set motor_pwm_protocol = DSHOT600`

Reply:
hi ，I have corrected the configuration file according to the use of hardware, please check them, thank you!